### DISCUSSION TITLE
What country is this?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#513] [Galaxy Trivia] [answered]
**LAAPEC**
**OTRWE**
**APRSI**
**AONTSCIRS**

*answer using country flag emoji*

**Comment by peterbaccaro** [comment, marked as answer]
🇫🇷 - FRANCE

The anagrams were:
- PALACE
- TOWER
- PARIS
- CROISSANT

**Comment by thuongtruong1009** [comment]
LAAPEC: Palace
OTRWE: TOWER
APRSI: Paris
AONTSCIRS: Costa Rica 🇨🇷
So, the country flags corresponding to the anagrams are:

🇳🇴 Norway
🇨🇷 Costa Rica
fr France
